mozci/data/sources/activedata/__init__.py

```diff
@@ -20,7 +20,7 @@ class ActiveDataSource(DataSource):
     name = "adr"
     supported_contracts = (
         "push_tasks",
-        "push_tasks_results",
+        # "push_tasks_classifications",
```

Comment:
I guess this was commented out for testing purposes?

Reply:
Yep, thanks for catching!